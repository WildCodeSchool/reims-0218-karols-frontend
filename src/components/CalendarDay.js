import React from "react"

const CalendarDay = () => {
  return (
    <div className="text-center">
      <h5>Vendredi</h5>
      <h5>25 mai</h5>
    </div>
  )
}

export default CalendarDay
