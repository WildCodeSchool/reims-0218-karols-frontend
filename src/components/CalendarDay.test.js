import React from "react"
import ReactDOM from "react-dom"
import CalendarDay from "./CalendarDay"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<CalendarDay />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<CalendarDay />).toJSON()
  expect(tree).toMatchSnapshot()
})
