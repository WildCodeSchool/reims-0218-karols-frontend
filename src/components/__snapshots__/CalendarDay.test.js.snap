// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="text-center"
>
  <h5>
    Vendredi
  </h5>
  <h5>
    25 mai
  </h5>
</div>
`;
